lib/whitehall_importer.rb

```diff
@@ -1,6 +1,35 @@
 # frozen_string_literal: true
 
 module WhitehallImporter
+  def self.create_migration(organisation_content_id, document_type, items_per_page = 100)
+    whitehall_migration = WhitehallMigration.create!(organisation_content_id: organisation_content_id,
+                               document_type: document_type,
+                               start_time: Time.current)
+
+    page_number = 1
+    loop do
+      puts "Loading page #{page_number}"
```

Comment:
We should reserve any puts to be done in the rake task.

Reply:
Sure, removed.  This was leftover from moving the code out of the rake task.